modules/luci-base/htdocs/luci-static/resources/form.js

```diff
@@ -1892,6 +1892,26 @@ var CBIAbstractValue = CBIAbstractElement.extend(/** @lends LuCI.form.AbstractVa
 		return true;
 	},
 
+	/**
+	 * Apply custom tooltip.
+	 *
+	 * This method is invoked on form render.
+	 *
+	 * The default implementation of this method does nothing and always
+	 * returns `false`. User code may overwrite this method to provide
+	 * additional tooltip information.
+	 *
+	 * @abstract
+	 * @param {string} section_id
+	 * The configuration section ID
+	 *
+	 * @returns {*}
+	 * The method shall return a string. This is show as an tooltip.
+	 */
+	tooltip: function(section_id) {
+		return false;
+	},
```

Comment:
I think we should allow this property to be either a string (in which case it'll be filtered through `String.format()` with section_id as argument) or a function (which behaves like you implemented it) - similar to e.g. [`TypedSection.addbtntitle`](http://openwrt.github.io/luci/jsapi/LuCI.form.TypedSection.html#addbtntitle)

The vast majority of tooltip using code likely only provides a single static value.